Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/MainAddActionsNavigationPage.xaml.cs

```diff
@@ -41,11 +45,12 @@ public MainAddActionsNavigationPage(Context context)
             xNavigationBarPnl.Visibility = Visibility.Collapsed;
             xSelectedItemFrame.ContentRendered += NavPnlActionFrame_ContentRendered;
             SetRecordButtonAccessebility();
+            xApplicationModelsPnl.Visibility = Visibility.Collapsed;
         }
 
         private void SetRecordButtonAccessebility()
         {
-            Agent mAgent = AgentHelper.GetDriverAgent(mContext.BusinessFlow.CurrentActivity, mContext.Runner, mContext);
+            mAgent = AgentHelper.GetDriverAgent(mContext.BusinessFlow.CurrentActivity, mContext.Runner, mContext);
```

Comment:
why we still have this  mAgent = AgentHelper.GetDriverAgent... Agent now exist on Context

Reply:
Updated with ContextProperty change